src/mainboard/clevo/adl-p/vboot-rwa.fmd

```diff
@@ -1,7 +1,8 @@
 FLASH 32M {
 	SI_ALL 5M {
 		SI_DESC 4K
-		SI_ME
+		SI_EC 128K
+		SI_ME 4820K
```

Comment:
```suggestion
		SI_ME
```

Let it occupy the rest of space.

Reply:
Fixed in https://github.com/Dasharo/coreboot/pull/369/commits/c4ed91025ad83b80e5a723455745f8330115f4f1